client/packages/system/src/Asset/utils.ts

```diff
@@ -0,0 +1,23 @@
+import { LocaleKey, TypedTFunction } from '@common/intl';
+import { AssetCatalogueItemFragment } from './api';
+import { Formatter } from '@common/utils';
+
+export const assetCategoryListItemsToCsv = (
+  items: AssetCatalogueItemFragment[],
+  t: TypedTFunction<LocaleKey>
+) => {
+  const fields: string[] = [
+    'id',
+    t('label.code'),
+    t('label.manufacturer'),
+    t('label.model'),
+  ];
+
+  const data = items.map(node => [
+    node.id,
+    node.code,
+    node.manufacturer,
+    node.model,
```

Comment:
Maybe add a TODO to update to include type, class, etc after we complete https://github.com/msupply-foundation/open-msupply/issues/3103 ?